### PR TITLE
EZP-29993: As the eZ Platform developer I would like to use Sass compiler in the Admin UI modules styles

### PR DESCRIPTION
**Pull request by sunpietro**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29993
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

### CODE REVIEW

File: src/bundle/Resources/public/scss/_icons.scss [hunk 1]
```diff
@@ -7,11 +7,26 @@
         height: calculateRem(16px);
     }
 
+    &--small-medium {
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_icons.scss]: I think that `medium-small` would be more pronounceable.

File: src/bundle/Resources/public/scss/_icons.scss [hunk 2]
```diff
@@ -7,11 +7,26 @@
...
+        width: calculateRem(20px);
+        height: calculateRem(20px);
+    }
+
     &--medium {
         width: calculateRem(24px);
         height: calculateRem(24px);
     }
 
+    &--large {
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_icons.scss]: I wonder if `48px` and `64px` are based on elements from the interface?

File: src/bundle/Resources/public/scss/_icons.scss (hunk 1 above)
Comment (sunpietro) [reply src/bundle/Resources/public/scss/_icons.scss]: Me and @dew326 agreed to use this name

File: src/bundle/Resources/public/scss/_icons.scss (hunk 2 above)
Comment (sunpietro) [reply src/bundle/Resources/public/scss/_icons.scss]: Yes, these sizes come from AdminUI modules.

Comment (tischsoic) [reply src/bundle/Resources/public/scss/_icons.scss]: Great!

**Pull request merged**